tutorials/xr/setting_up_xr.rst

```diff
@@ -93,6 +94,36 @@ Next we need to add a script to our root node. Add the following code into this
         else:
             print("OpenXR not initialised, please check if your headset is connected")
 
+  .. code-tab:: csharp
+    using Godot;
+    using System;
+
+    public partial class MyNode3D : Node3D
+    {
+        private XRInterface _xrInterface;
+
+        public override void _Ready()
+        {
+            xrinterface = XRServer.FindInterface("OpenXR");
+            if(xrinterface != null && xrinterface.IsInitialized())
```

Comment:
Since the member was renamed.

```suggestion
            _xrInterface = XRServer.FindInterface("OpenXR");
            if(_xrInterface != null && _xrInterface.IsInitialized())
```